level2/loadbalancer.tf

```diff
@@ -0,0 +1,7 @@
+module "lb" {
+  source = "../modules/lb"
+
+  env_code  = var.env_code
+  vpc_id    = data.terraform_remote_state.level1.outputs.vpc_id
+  subnet_id = data.terraform_remote_state.level1.outputs.public_subnet_id
+}
```

Comment:
could you please, add a newline at the end of file.

Reply:
Done.